sos/report/plugins/oracle.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import os
+from sos.report.plugins import Plugin, RedHatPlugin, PluginOpt
+
+
+class Oracle(Plugin, RedHatPlugin):
```

Comment:
Is there a more specific component name to use here perhaps? I think there may be some ambiguity with this and Oracle Linux as a distribution, or the Oracle DB offering, etc...

Since the plugin is centered around `oratab`, would that make sense as a name? I'll admit I'm not familiar with what the component/service surrounding oratab is, so maybe that doesn't fit.